sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DescribeTableExec.scala

```diff
@@ -42,6 +42,7 @@ case class DescribeTableExec(
     addClustering(rows)
 
     if (isExtended) {
+      addTableConstraints(rows)
```

Comment:
As per the SPIP https://docs.google.com/document/d/1EHjB4W1LjiXxsK_G7067j9pPX0y15LUF1Z5DlUPoPIo/edit?tab=t.0, the constraint info should be at the end of the output